sdk/lib/crt/math/libm_sse2/fmod.asm

```diff
@@ -142,7 +142,7 @@ fmod		PROC	FRAME
 		jmp	@fail
 
 @fail:
-		lea	rcx, [@fmodz]		; fname
+		lea	rcx, @fmodz		; fname
```

Comment:
All the other asm files from the converted stuff are left unchanged?

Reply:
This is 3rd-party code. Add a ReactOS comment?

Reply:
@HBelusca Yes, they are left unchanged, because there is nothing to fix.
@SergeGautherie Yes, it is 3rd party code, but it already had a number of fixes and I don't see any point in adding comments. It's in the git history.